redhat-cop/infra-ansible
# Updated Satellite role to support manifest, repos and activation keys

### What does this PR do?
Introducing support for a manifest, repos (with a sync plan) and activation keys as part of the Satellite role. 

### How should this be tested?
Set the correct inventory in the `tests` directory and run the `test.yml` playbook. See the role's README for more details as well.

### Is there a relevant Issue open for this?
N/A

### People to notify
cc: @day4skiing
